### DISCUSSION TITLE
Oracle user info

### DISCUSSION THREAD

**Discussion by Jasonlawsondukeenergy** [dbeaver/dbeaver#21556] [Q&A]
### Description

DBeaver does not accurately display Oracle User information.  For example, the Lock date:
Security->Users->[user id]->View User
 
Select u.*, v.*
from sys.DBA_USERS u, sys.v_$pwfile_users v
where u.username = 'MDMCUBE'
and u.username = v.username(+);
![dbeaver lock date bug](https://github.com/dbeaver/dbeaver/assets/148122257/949efcbb-d7d2-409f-82c9-a50fdcc076c4)

### DBeaver Version

23.1.2

### Operating System

Windows 10 Enterprise

### Database and driver

Oracle
oracle.jdbc.OracleDriver

### Steps to reproduce

1. connect to an Oracle database
2. Navigate to Security->Users->[choose a locked user id]->View User
3. Notice the Lock Date is not filled
4. execute the query below for the same user id
Select u.*, v.*
from sys.DBA_USERS u, sys.v_$pwfile_users v
where u.username = :lockeduser
and u.username = v.username(+);

**Comment by LonwoLonwo** [comment]
Hello @Jasonlawsondukeenergy 

Could you please try to turn on this setting and recheck your User tab info?

![2023-10-17 12_10_15-Connect to a database](https://github.com/dbeaver/dbeaver/assets/45152336/d9648a13-defc-4adb-acbb-f4adf8a5952c)

**Comment by Jasonlawsondukeenergy** [comment]
Thank you for the quick response!  That resolved the issue.
-Jason
